pages/_app.tsx

```diff
@@ -43,15 +44,17 @@ function MyApp({ Component, pageProps }: AppProps) {
       <RainbowKitProvider modalSize="compact" chains={chains}>
         <ThemeProvider>
           <UserProvider>
-            <SessionProvider>
-              <AdminProvider>
-                <ChatProvider>
-                  <Component {...pageProps} />
-                  <ToastContainer />
-                  <Analytics />
-                </ChatProvider>
-              </AdminProvider>
-            </SessionProvider>
+            <MintProvider>
```

Comment:
MintProvider can be limited to the `/mint` route right?

do we need to wrap the base `_app`?